src/app/components/core/north/north-task-modal/north-task-modal.component.css

```diff
@@ -1,3 +1,8 @@
 .box {
   box-shadow: 0 0 0 0 !important;
 }
+
+label{
+  width: 10vw !important;
```

Comment:
app-view-config-item have this? if its for wizard parent component then we need to put this with some custom class, as we don't want to ellipsized config name|key